app/routers/book.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import APIRouter, HTTPException, Depends
+from sqlalchemy.orm import Session
+from app.models import Book as SQLAlchemyBook
+from app.schemas import Book as PydanticBook
+from app.database import get_db
+
+router = APIRouter()
+
+
+@router.get("/{book_id}/availability", response_model=PydanticBook)
+async def check_availability(book_id: int, db: Session = Depends(get_db)):
+    db_book = db.query(SQLAlchemyBook).filter(SQLAlchemyBook.book_id == book_id).first()
```

Comment:
We have crud.py file to execute database operations. So, we shouldn't write this operation here.